source/main.py

```diff
@@ -1,30 +1,192 @@
 import sys
-from PyQt5.QtWidgets import QWidget, QDesktopWidget, QApplication
+from PyQt5 import QtCore, QtGui, QtWidgets
+from PyQt5.QtWidgets import QApplication, QMainWindow, QWidget, QPushButton
 
+class Ui_MainWindow(object):
```

Comment:
@mandelshtamd , 

could you move this code from main.py to some other file?

Usually `main` is responsible only for launching. You could have a package `view` with all the code responsible for rendering.